src/ioptions.ts

```diff
@@ -30,4 +30,5 @@ export interface IOptions
 	tsconfigDefaults: any;
 	sourceMapCallback: (id: string, map: string) => void;
 	objectHashIgnoreUnknownHack: boolean;
+	extraCacheKeys: string[]
```

Comment:
```suggestion
	extraCacheKeys: any
```

This can contain literally any variables etc to add to the hash. It can be a single var, an object, an array, etc. Doesn't have to be only strings either.